src/pages/Stønadoversikt/utils.ts

```diff
@@ -146,8 +145,6 @@ export const utledKolonnebredde = (beløp: number) => {
     return '2.85rem';
   } else if (beløp < 10000) {
     return '3.65rem';
-  } else if (beløp < 100000) {
-    return '4.2rem';
   }
-  return '4.8rem';
+  return '5.5rem';
```

Comment:
Uten endringen la beløp og nok seg på to linjer. Kanskje dette kan løses på en annen måte?

Reply:
Hvordan ser mobilvisningen ut med den bredden? Kan være lurt å passe på at man ikke må scrolle til høyre, iallefall 😬 

Husker det var litt knot å få dette mobilvennlig. (Gjelder særlig for de som har samordningsfradrag)

Reply:
Det går greit, på mindre skjermer blir det ikke vist som kolonne, men som vist ved å trykk på mer på cellen.